inc/endpoints/class-data-endpoint.php

```diff
@@ -41,15 +42,29 @@ public function register_rest_routes() {
 			return;
 		}
 
-		foreach ( $data_endpoints as $endpoint ) {
-			register_rest_route(
-				self::get_namespace(),
-				'/data/' . $endpoint['slug'],
+		// Register each endpoint.
+		foreach ( $data_endpoints as $args ) {
+
+			$args = wp_parse_args(
+				$args,
 				[
-					'methods'  => \WP_REST_Server::READABLE,
-					'callback' => $endpoint['callback'],
+					'callback'            => '__return_true',
+					'methods'             => \WP_REST_Server::READABLE,
+					'permission_callback' => '__return_true',
+					'slug'                => '',
 				]
 			);
+
+			// Ensure we have a slug.
+			if ( empty( $args['slug'] ) || ! is_string( $args['slug'] ) ) {
+				break;
```

Comment:
This should be `continue;` otherwise, other endpoints in the array won't get registered.